NST/src/main/java/com/ebay/service/logger/platforms/HarLogger.java

```diff
@@ -27,7 +27,7 @@ public Platform getPlatformAssociation() {
 	public void writeMocks(List<ServiceCallCacheData> calls, String testClassName, String testMethodName) {
 		
 		FormatWriterUtil.removeMockFilesMatchingClassAndMethodName(testClassName, testMethodName);
-		int sequenceCounter = 0;
+		int sequenceCounter = 1;
```

Comment:
Why was this changed?

Reply:
Nvm, I see it, might cause some issue with our request index validator on iOS, but its ok